src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -161,6 +164,12 @@ def load_arguments(self, _):
         c.argument('encryption_algorithm', arg_type=get_enum_type(JsonWebKeyEncryptionAlgorithm),
                    arg_group='Customer Managed Key', help='Algorithm identifier for encryption.')
 
+        # Kafka Rest Proxy
+        c.argument('kafka_client_group_id', arg_group='Kafka Rest Proxy',
```

Comment:
Is it possible to provide short name for your argument here?

Reply:
> Is it possible to provide short name for your argument here?

Hi @Juliehzl , Thanks. This is the shortest name that can explain the meaning. And there are already some properties' name are longer than this one. For example : `workernode_data_disk_storage_account_type=None`, `workernode_data_disk_size=None`,
We should consider the meaning firstly, right?

Reply:
You can provide `options_list=['--kafka-client-group-id', '-k']` to give both descriptive name and short name. (Feel free to find a vaild short name here)
The same for the following one.